airflow/www/utils.py

```diff
@@ -282,7 +289,18 @@ def dt(attr):
         f = f.isoformat() if f else ''
         if timezone.utcnow().isoformat()[:4] == f[:4]:
             f = f[5:]
-        return Markup("<nobr>{}</nobr>").format(f)
+        return Markup('<time datetime="{}">{}</time>').format(f, f)
+    return dt
+
+
+def localized_f(attr_name):
+    def dt(attr):
+        f = attr.get(attr_name)
+        if f is None:
+            return Markup('<time datetime="{}">{}</time>').format(f, f)
+        else:
+            f = f.strftime(date_format)
+            return Markup('<time datetime="{}">{}</time>').format(f, f)
```

Comment:
I would have though that we'd want the datetime attribute to always be in ISO8601, and convert just on client side.

The main reason is that server locales are quite likely to be either C, or `en_US.UTF-8` (as this is the default for many cloud images, on EC2/GCE etc.).

Reply:
Everything in this PR is converting client-side only from whatever format is coming from the server. I'm not too familiar with Python so if there's a better func to call to induce the formatting, I'd love to learn!